staging/src/k8s.io/code-generator/cmd/client-gen/generators/generator_for_clientset.go

```diff
@@ -144,12 +148,23 @@ func NewForConfig(c *$.Config|raw$) (*Clientset, error) {
 		}
 		configShallowCopy.RateLimiter = $.flowcontrolNewTokenBucketRateLimiter|raw$(configShallowCopy.QPS, configShallowCopy.Burst)
 	}
-	factory, err := rest.RESTClientFactoryFor(&configShallowCopy)
+	factory, err := $.RESTClientFactoryFor|raw$(&configShallowCopy)
 	if err!=nil {
 		return nil, err
 	}
+	options := []$.RESTClientOption|raw${}
+	if configShallowCopy.RateLimiter != nil {
+		options = append(options, $.RESTClientWithRateLimiter|raw$(configShallowCopy.RateLimiter))
```

Comment:
the problem is that this is sharing the same ratelimiter :/, before each clienset created its own